tmt/convert.py

```diff
@@ -46,6 +46,116 @@ def repr_str(dumper, data):
 #  Convert
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+def read_manual(plan_id, case_id, disabled):
+    """
+    Reads metadata of manual test cases from Nitrate
+    """
+    try:
+        import nitrate
+    except ImportError:
+        raise ConvertError('Install nitrate module to import metadata.')
+    # Turns off nitrate caching
+    nitrate.set_cache_level(0)
+
+    try:
+        tree = fmf.Tree(os.getcwd())
+    except fmf.utils.RootError:
+        raise ConvertError("Initialize metadata tree using 'tmt init'")
+
+    try:
+        if plan_id:
+            all_cases = nitrate.TestPlan(int(plan_id)).testcases
+            case_ids = [case.id for case in all_cases if case.manual]
+        else:
+            case_ids = [int(case_id)]
+    except ValueError:
+        raise ConvertError('Value of plan/case option must be an integer')
+
+    # Create directory to store manual tests in
+    old_cwd = os.getcwd()
+    os.chdir(tree.root)
+    try:
+        os.mkdir('Manual')
```

Comment:
Let's use lowercase to keep consistency with other templates.

Reply:
I chose uppercases to keep consistency with other dist git test directories (Regression, Sanity...)

Reply:
I see. I believe in the near future we will/should get rid of this old classification which IMO does not make any sense. But those old directories will be with us for some time. And users are expected to rename / move imported tests around as necessary. So I leave the naming up to you.